src/Sql/Sql/ManagedInstance/Cmdlet/SetAzureSqlManagedInstance.cs

```diff
@@ -178,6 +179,24 @@ public class SetAzureSqlManagedInstance : ManagedInstanceCmdletBase
         [Parameter(HelpMessage = "Skip confirmation message for performing the action")]
         public SwitchParameter Force { get; set; }
 
+        /// <summary>
+        /// Gets or sets the instance compute generation
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The compute generation for the instance.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter(Constants.ComputeGenerationGen5)]
+        public string ComputeGeneration { get; set; }
+
+        /// <summary>
+        /// Gets or sets the instance SKU name
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The SKU name for the instance e.g. 'GP_Gen4','BC_Gen4'.")]
```

Comment:
I suggest adding 'GP_Gen5','BC_Gen5' in help message as Gen4 is becoming deprecated

Reply:
Nice catch. Fixed